src/D2L.CodeStyle.TestAnalyzers/IgnoreAttribute/IgnoreAttributeAnalyzer.cs

```diff
@@ -41,6 +42,15 @@ internal sealed class IgnoreAttributeAnalyzer : DiagnosticAnalyzer {
 
 			foreach( var attribute in root.Attributes ) {
 				if( attribute.Name.ToString().Equals( "Ignore" ) ) {
+					var parentClassDeclaration = root.AncestorsAndSelf().OfType<ClassDeclarationSyntax>().ToImmutableArray().Single();

```

Comment:
Sorry, I was confusing. I think just make sure that the correct attribute is being inspected. 
I.e., `NUnit.Framework.IgnoreAttribute`. The conflict you're getting with `ReflectionSerializer` is a different `Ignore`.